.vscode/settings.json

```diff
@@ -55,6 +55,6 @@
 		"streambuf": "cpp",
 		"thread": "cpp",
 		"typeinfo": "cpp",
-		"future": "cpp"
+		
```

Comment:
"future" was in the file before. please leave it there

Note, that I do not know where exactly this settings.json file is used (I only have a faint idea). I will look into it for later development.

Reply:
done